public/electron.js

```diff
@@ -41,9 +41,13 @@ const isMac = process.platform === 'darwin';
 
 function createWindow() {
   const framelessConfig = isMac ? { titleBarStyle: 'hidden' } : { frame: false };
+  const iconPath = path.join(
+    process.env.NODE_ENV === 'production' ? process.env.PUBLIC_URL : __dirname,
```

Comment:
There's an `isDev` variable already, we should use that here (unless there's a good reason not to).